examples/active-active/variables.tf

```diff
@@ -0,0 +1,53 @@
+variable "aws_role_arn" {
```

Comment:
Part of the task's Acceptance Criteria is `Consistent alphabetization for variables (Grouping of variables to their relevant sections)`, so this file needs to be updated.

Reply:
vars are alphabetically sorted now without sections